### DISCUSSION TITLE
Getting a 'move out of range' error when starting print

### DISCUSSION THREAD

**Discussion by peterdijkstra** [bassamanator/Sovol-SV06-firmware#15] [Q&A]
Hi, 3d printing newbie here!

First of all, thanks for taking the time to make this!

Been having a lot of fun with my first 3d printer, the sv06 on stock firmware, but I wanted to take the next step and install Klipper. I've followed the README.md, have everything setup, and everything seems to be working fine.. except for printing.

I have tried feeding it two different gcodes that I had SuperSlicer export using the included config bundle. A bed leveling test and a calibration cube. I keep getting the following error, after the bed and extruder are heated up and it does the home:

`Move out of range: 84.500 275.000 30.400 [67.000]`

After that it stops the print process.

I suppose `275` is bigger than `stepper_y.position_max`'s `223`. But I'm not sure where it would get that big of a number from.

I am running Klipper on a Raspberry Pi 4.

This might be better suited for issues instead of discussions, but I'm inclined to think this a user error considering I'm new to this so I will leave it here for now.

Any help or direction would be greatly appreciated :-).

**Comment by bassamanator** [comment]
As a first step, let's just home all axis, move the printer to x=223, y=223, z=150.

Are you able to do this in mainsail/fluiddpi?

**Reply by peterdijkstra** [reply]
Yes that works fine.

Using mainsail fwiw

**Comment by peterdijkstra** [comment]
Well, it seems to working now after being powered down for the night.

I did try to restart the firmware and klipper and all that last night when it wasn't working, but I did not try fully powering down both the Pi and the sv06.

In any case, thanks Bassam for taking the time to reply!
